src/components/pages/Home/RenderHomePage.js

```diff
@@ -35,7 +35,7 @@ import AddFaq from '../AdminTools/AddFaq';
 import ManageFaqPage from '../AdminTools/ManageFaq';
 import EditFaqPage from '../AdminTools/EditFaq';
 
-// self-import font for MUI components
+// Global styling for MUI components
```

Comment:
Yes! I love to see this here so we can reuse code.

Reply:
Seconding this, it's definitely a nice-to-have!